src/gpu/nodeKernels.cu

```diff
@@ -43,6 +43,12 @@ __global__ void kTransformRays(size_t rayCount, const Mat3x4f* inRays, Mat3x4f*
 	outRays[tid] = transform * inRays[tid];
 }
 
+__global__ void kVelocityDistortRays(size_t rayCount, const Mat3x4f* inRays, const float* offsets, size_t offsetsCount, const Vec3f sensorLinearVelocity, const Vec3f sensorAngularVelocity, Mat3x4f* outRays)
+{
+	LIMIT(rayCount);
+	outRays[tid] = Mat3x4f::TRS((offsets[tid] * (sensorLinearVelocity * 0.001f)) * 1000.0f, (offsets[tid] * (sensorAngularVelocity) * 0.001f) * 1000.0f, Vec3f(1.0f, 1.0f, 1.0f)) * inRays[tid];
```

Comment:
Please put a comment that describes why we are performing unit conversions in this way.